flexget/plugins/input/anidb_feed.py

```diff
@@ -0,0 +1,453 @@
+                valid_only: yes
+
+            valid_only: yes
+            # If enabled only 'valid' entries are returned, that have a size, release group, resolution and source field
+            # This happens after the file was added to anidb and has passed some initial tests/parsing
+
+            priority: (all, medium-high, high)
+            # defaults to 'all', which are the anidb notification priorities
+
+            include: ('special', 'op-ending', 'trailer-promo')
+            # Also report none regular feed file types, by default any none regular file is skipped.
+            anidb_feed:
+                url: http://anidb.net/perl-bin/animedbfeed.pl?id=xxxxx................
+                include: special
+
+        Adds those extra Entry fields if found:
+        'content_size'
+        'rss_pubdate'
+        'anidb_fid'             # anidb file_id
+        'anidb_file_version'    # fileversion aka v1-5
+        'anidb_file_crc32'      # the anidb validated hash for this file relese
+        'anidb_feed_added'
+        'anidb_feed_source'
+        'anidb_feed_resolution'
+        'anidb_feed_crc_status'
+        'anidb_feed_language'
+        'anidb_feed_group'
+        'anidb_feed_grouptag'
+        'anidb_feed_subtitle_language'
+        'anidb_feed_priority'
+        'anidb_feed_quality'
+        'anidb_feed_size'
+    """
+
+    schema = {
+        'type': 'object',
+        'properties': {
+            'url': {'type': 'string', 'format': 'url'},
+            'priority': {'type': 'string', 'enum': ['all', 'medium-high', 'high'], 'default': 'all'},
+            'valid_only': {'type': 'boolean', 'default': False},
+            'include': one_or_more({'type': 'string', 'enum': ANIDB_EXTRA_TYPES}, unique_items=True)
+        },
+        'additionalProperties': False,
+        'required': ['url'],
+    }
+
+    def prepare_config(self, config):
+        if 'include' in config and not isinstance(config['include'], list):
+            config['include'] = [config['include']]
+        return config
+
+    # anidb notification update interval is 15 minutes
+    @cached('anidb_feed', persist='15 minutes')
+    def on_task_input(self, task, config):
+        config = self.prepare_config(config)
+        task.requests.add_domain_limiter(LIMITER)
+        # Create entries by parsing AniDB feed
+        log.verbose('Retrieving AniDB feed')
+        url = re.sub(r'&pri=[1-9]', '', config['url'])  # strip existing priority
+        priority_fragment = ''
+        if 'animedbfeed' in url:
+            priority = 0
+            if config['priority'] == 'medium-high':
+                priority = 1
+            elif config['priority'] == 'high':
+                priority = 2
+            priority_fragment = '&pri=%s' % priority
+        return self.fill_entries_for_url(url + priority_fragment, task, config)
+
+    @staticmethod
+    def is_valid_entry(entry, ns_prefix_name=''):
```

Comment:
I guess so i can directly do this, without checking `ns_prefix_name`
`key_name = ns_prefix_name + key`
Its funny the parameter actually was named `namespace_prefix_name`, but i felt it was too long if used via `is_valid_entry(entry, namespace_prefix_name="mynamepsace")`